### DISCUSSION TITLE
There is a Discord Server for Discord app or it's only in WidgetBot?

### DISCUSSION THREAD

**Discussion by Azmiko46** [LizardByte/community#12] [Sunshine] [answered]
Hi, there is a Discord Server for Discord app or it's only in WidgetBot?
Thanks, I wait your response.

**Comment by ReenigneArcher** [comment]
Login and the server will be added automatically.

**Reply by Azmiko46** [reply]
I logged it and the server doesn't add me in the discord app.

**Reply by ReenigneArcher** [reply]
> I logged it and the server doesn't add me in the discord app.

Interesting... what is your discord username?

**Reply by Azmiko46** [reply]
Azmiko46#8376

**Reply by ReenigneArcher** [reply]
Can you try sending a message there after you login?

**Reply by Azmiko46** [reply]
Error sending message
Error: This guild doesn't have guestmode enabled.

**Reply by ReenigneArcher** [reply]
Okay... You can join the server here: https://sunshinestream.github.io/discord_join/

**Reply by Azmiko46** [reply]
Okay, thx very much!

**Comment by ReenigneArcher** [comment, marked as answer]
I also made an adjustment to widgetbot settings, hopefully that will allow people to join from there.

**Reply by ReenigneArcher** [reply]
I believe this is working as intended now. A user using widgetbot will appear as a `BOT`, but now they are also be added to the server list of users.
![image](https://user-images.githubusercontent.com/42013603/155740485-9571b24e-7389-44f5-9b94-a2eb2067fb5f.png)

**Reply by ReenigneArcher** [reply]
The widget also has a "join" button at the top which will just the discord invite link like normal.
